sdk/monitor/monitor-query/src/timespanConversion.ts

```diff
@@ -21,6 +21,22 @@ export function convertTimespanToInterval(timespan: TimeInterval): string {
   throw new TypeError("Invalid Timespan - no valid fields assigned.");
 }
 
+export function convertIntervalToTimeIntervalObject(timespan: string): TimeInterval {
```

Comment:
We definitely need unit test coverage for this - every possibility here can be easily unit-testable

Reply:
Yes I added to the issue I made for the other tests.